Description: Rename SNMP.3 to SNMP.3pm
 The upstream build system installs the perl SNMP man page as SNMP.3
 While the Debian dh_install renames it, it causes the file to be put into
 libsnmp-dev not libsnmp-perl package.
 Giving it the correct name in the first place means its picked up by the
 correct package.
Author: Craig Small <csmall@debian.org>
Origin: Debian
Bug-Debian: https://bugs.debian.org/969436
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2021-09-20
--- a/perl/SNMP/Makefile.PL
+++ b/perl/SNMP/Makefile.PL
@@ -25,7 +25,7 @@
     my %Params = (
 		  NAME         => 'SNMP',
 		  dist         => { SUFFIX => "gz", COMPRESS => "gzip -9f"},
-		  MAN3PODS => { 'SNMP.pm' => '$(INST_MAN3DIR)/SNMP.3' },
+		  MAN3PODS => { 'SNMP.pm' => '$(INST_MAN3DIR)/SNMP.3pm' },
 		  XSPROTOARG   => '-noprototypes', 	# XXX remove later?
 		  VERSION_FROM => 'SNMP.pm',
 		  realclean        => { FILES => 'host' },
